spec/ibandit/pseudo_iban_assembler_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+describe Ibandit::PseudoIBANAssembler do
+  subject(:assembler) { described_class.new(local_details) }
+
+  describe '#assemble' do
+    subject(:pseudo_iban) { assembler.assemble }
+
+    context 'for Sweden' do
+      context 'with valid parameters' do
+        let(:local_details) do
+          {
+            country_code: 'SE',
+            branch_code: '1281',
+            account_number: '0105723'
+          }
+        end
+
+        it { is_expected.to eq('SEZZZ1281ZZZ0105723') }
+      end
+
+      context 'without a branch code' do
+        let(:local_details) do
+          {
+            country_code: 'SE',
+            account_number: '0105723'
```

Comment:
Do we want to support the case where clearing number is entered with the account number?